setup_utils/fsd50k_to_dcase_format.py

```diff
@@ -0,0 +1,32 @@
+import os
+import shutil
+
+# Path to the DCASE .txt containing directory structure
+FSD50K_DCASE_DIR_STRUCT_TXT = "/scratch/data/FSD50K/FSD50K_DCASE/FSD50K_selected.txt"
```

Comment:
where can we find this file?

Reply:
It is defined within the README.

I think we should start by merging this and proceed to the other PRs (this one is the smaller one)